pkgs/games/hikounomizu/default.nix

```
@@ -0,0 +1,77 @@
+{ lib
+, stdenv
+, fetchurl
+, cmake
+, pkg-config
+, util-linux
+, libGL
+, freetype
+, pugixml
+, SDL2
+, SDL2_image
+, openal
+, libogg
+, libvorbis
+, libGLU
+, synfigstudio
+, inkscape
+, imagemagick
+, pngquant
+, xz
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hikounomizu";
+  version = "0.9.2";
+
+  src = fetchurl {
+    url = "http://download.tuxfamily.org/hnm/${version}/hikounomizu-${version}-src.tar.bz2";
+    hash = "sha256-ZtvzQAiYG4IcdgKiBDIQFOJVnLbz1TsiIbdZr/0Y2U8=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    # for make data
+    util-linux
+    synfigstudio.synfig
+    inkscape
+    imagemagick
+    pngquant
+    xz
+  ];
+
+  buildInputs = [
+    libGL
+    freetype
+    pugixml
+    SDL2
+    SDL2_image
+    openal
+    libogg
+    libvorbis
+    libGLU
+  ];
+
+  postBuild = ''
+    make data
```

```suggestion
    make data -j$NIX_BUILD_CORES
```
or is that sequential?